### ISSUE TITLE
Error on vitest 

### ISSUE THREAD

**Issue by adamtomi** [evanderkoogh/do-taskmanager#3]
Hi, I keep getting the following error when running vitest (**v0.23.4**) tests that involve durable objects that are using do-taskmanager. 

I'll have a fix that seems to mitigate this problem, I'll open a PR for it.
```
 ❯ packageEntryFailure ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:34838:11
Error: Failed to resolve entry for package "do-taskmanager". The package may have incorrect main/module/exports specified in its package.json.
 ❯ resolvePackageEntry ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:34835:5
 ❯ tryNodeResolve ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:34583:20
 ❯ Context.resolveId ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:34356:28
 ❯ Object.resolveId ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:41030:46
 ❯ TransformContext.resolve ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:40799:23
 ❯ normalizeUrl ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:37580:34
 ❯ TransformContext.transform ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:37719:47
 ❯ Object.transform ../../../../../../../D:/Dev/web/accessory/node_modules/vite/dist/node/chunks/dep-557f29e6.js:41102:30
```

**Comment by evanderkoogh** [comment]
Can you confirm this is fixed now that we have shipped your PR in `2.0.0-rc0`?

Thanks :)

**Comment by adamtomi** [comment]
Hey!

Thanks for merging my PR! I've done some testing and it seems this problem has been fixed. 

Thank you! :)
